### PR TITLE
fix f1score

### PR DESCRIPTION
**Pull request by wdevazelhes**
Hi @TheoGuyard , thanks for the awesome benchmark!

I think there's a typo in `f1score`, i.e. somehow the `w` and `w_true` are kept as such when I guess there should be some boolean conversion at some point. Here is a proposed fix, in case it helps!

### CODE REVIEW

**Comment by mathurinm** [comment]
@wdevazelhes thanks, to be even more robust I'd even call `sklearn.metrics.f1_score(w_true != 0, w != 0)` instead of using this function

**Comment by wdevazelhes** [comment]
sorry, just realized the import should be coming from safe import, will modify it now

**Review by tomMoral — APPROVED** [review]
Looks a good idea :)

File: benchmark_utils/metrics.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 with safe_import_context() as import_ctx:
     import numpy as np
+    from sklearn.metrics import f1_score
```
Comment (tomMoral) [inline benchmark_utils/metrics.py:5]: But if we depend on `scikit-learn`, we should update the `objective.py` file to add `requirements = ['scikit-learn']` to make the benchmark installable.

**Review by tomMoral — COMMENTED** [review]
Also, maybe it would make more sense to directly import the scikit-learn score function in the `objective.py` instead of having warppers

**Comment by wdevazelhes** [comment]
@tomMoral thanks! I just tried to implement that (sorry, I'm still a bit unfamiliar with the process as I'm just getting started with benchopt :p) 👍 🙏

**Review by tomMoral — APPROVED** [review]
This looks perfect!

Thanks a lot for your contribution 😁

**Comment by wdevazelhes** [comment]
@tomMoral @mathurinm thanks!🙏

**Comment by TheoGuyard** [comment]
Thanks @wdevazelhes !

**Pull request merged**